common/templates/errorDialog.modal.html

```diff
@@ -3,15 +3,17 @@
     <h2 class="modal-title">{{::ctrl.params.errorStatus}}</h2>
 </div>
 <div class="modal-body">
-  <span ng-bind-html="ctrl.params.message"></span>
-  <span ng-if="ctrl.params.showLogin"><a id="login-link" ng-click="ctrl.login()"> Please login</a> to continue.</span>
+    <span ng-bind-html="ctrl.params.message"></span>
+    <span ng-if="ctrl.params.showLogin"><a id="login-link" ng-click="ctrl.login()"> Please login</a> to continue.</span>
+    <span ng-if="ctrl.showDownloadPolicy">Download policy can be found <a id="download-policy-link" ng-href="{{ctrl.downloadPolicy}}">here</a>.</span>
```

Comment:
I only added this line, the other changes in this file were indenting from 2 spaces to 4.